README.md

```diff
@@ -4,13 +4,48 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+   ->  It ignores constants that can be significant. 
+
+   ->  Asymptotic doesn’t work well with for small input sizes, and therefore $n_0$ would be ignored. 
+
+   ->  The asymptotic analysis does not provide an exact running time.
```

Comment:
That's by definition, so not really misleading.

Reply:
Could that third reason, then be changed to "less-than or equal to " because in asymptotic analysis it gives the worst case limit and therefore, it could misleading.